backend/PennCourses/docs_settings.py

```diff
@@ -866,53 +866,86 @@ def _get_responses(self, path, method):
+        )
+
         if (
             path not in self.response_codes.keys()
             or method not in self.response_codes[path].keys()
             or not self.response_codes[path][method]
         ):
             if path in self.override_schema and method in self.override_schema[path].keys():
                 return {
+                    **{
+                        "200": {
+                            "description": "",
+                            **(
+                                {
+                                    "content": {
+                                        ct: {"schema": self.override_schema[path][method][200]}
+                                        for ct in self.response_media_types
+                                    }
+                                    if 200 in self.override_schema[path][method].keys()
+                                    else {
+                                        ct: {"schema": response_schema}
+                                        for ct in self.response_media_types
+                                    }
+                                }
+                            ),
+                        }
+                    },
+                    **auth_response_dict,
+                }
+            return {
+                **{
                     "200": {
+                        "content": {
+                            ct: {"schema": response_schema} for ct in self.response_media_types
+                        },
+                        # description is a mandatory property,
+                        # https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responseObject  # noqa: E501
+                        # TODO: put something meaningful into it
                         "description": "",
+                    }
+                },
+                **auth_response_dict,
+            }
+        if path in self.override_schema and method in self.override_schema[path].keys():
+            return {
+                **{
+                    str(k): {
+                        "description": v.replace("[SCHEMA]", ""),
                         **(
                             {
                                 "content": {
-                                    ct: {"schema": self.override_schema[path][method][200]}
+                                    ct: {"schema": self.override_schema[path][method][k]}
                                     for ct in self.response_media_types
                                 }
-                                if 200 in self.override_schema[path][method].keys()
+                                if k in self.override_schema[path][method].keys()
                                 else {
                                     ct: {"schema": response_schema}
                                     for ct in self.response_media_types
                                 }
                             }
+                            if "[SCHEMA]" in v
```

Comment:
This should be documented -- ideally there'd be a better way to do this than a string keyword, but this is fine as long as it's documented (both at the top [ where i think it is ] and right here.

Reply:
It is documented at the top, and I documented it here as well.